app/src/lib/api.ts

```diff
@@ -719,6 +754,33 @@ export class API {
     return await parsedResponse<IAPIRefStatus>(response)
   }
 
+  public async fetchPushControl(
```

Comment:
a little doc for this method please? i think its worth calling out that if it can't get the info, it defaults to assuming full access for the user (which i think is the right approach btw).